python/katana/local/_graph.pyx

```diff
@@ -275,6 +278,28 @@ cdef class GraphBase:
         """
         return str(self.underlying_property_graph().rdg_dir(), encoding="UTF-8")
 
+    @property
+    def node_types(self):
+        """
+        The types of atomic node types in the graph.
+
+        :rtype: list
+        """
+        cdef const EntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        type_ids = manager.GetAtomicEntityTypeIDs()
+        types = [EntityType.Make(manager, type_id) for type_id in type_ids]
+        return types
+
+    @property
+    def edge_types(self):
+        """
+        The types of atomic edge types in the graph.
+
+        :rtype: list
```

Comment:
There is a convention of writing `list[T]` to give the element type, so this can be `list[EntityType]`.